### PR TITLE
[JIRA-5] added goodbye.txt for say goodbye

### PR DESCRIPTION
**Pull request by luzperdomo92**
se agrego funcionalidad para el archivo goodbye

### CODE REVIEW

File: first_file.txt [hunk 1]
```diff
@@ -7,4 +7,7 @@ THIS CHANGE MAY BE USEFUL LATER
 console.log("something");
 THIS CHANGE MAY BE USEFUL LATER
 
+second line.
+agregue el archivo goodbye.txt
```
Comment (LSanclemente) [inline first_file.txt:12]: add before this "ADIOS AMIGO"

**Review by LSanclemente — CHANGES_REQUESTED** [review]

File: first_file.txt (hunk 1 above)
Comment (luzperdomo92) [reply first_file.txt:12]: added line requested

Comment (luzperdomo92) [reply first_file.txt:12]: added line requested

**Comment by LSanclemente** [comment]
please rebase before commit

**Review by LSanclemente — APPROVED** [review]

**Pull request merged**